.github/workflows/scripts/linux_arm64/build_catalyst.sh

```diff
@@ -0,0 +1,81 @@
+set -e -x
+cd /catalyst
+git config --global --add safe.directory '*'
+# Process args
```

Comment:
To make this and the other shell files easier to parse for my old eyes, can a newline be added after the end of each block?

Reply:
Sorry :) Done